hydro_deploy/hydroflow_plus_cli_integration/src/runtime.rs

```diff
@@ -3,38 +3,40 @@ use std::rc::Rc;
 
 use hydroflow_plus::lang::parse::Pipeline;
 use hydroflow_plus::node::{
-    HfCluster, HfClusterBuilder, HfDeploy, HfNode, HfNodeBuilder, HfSendManyToMany,
-    HfSendManyToOne, HfSendOneToMany, HfSendOneToOne,
+    ClusterBuilder, Deploy, HfCluster, HfNode, HfSendManyToMany, HfSendManyToOne, HfSendOneToMany,
+    HfSendOneToOne, NodeBuilder,
 };
 use hydroflow_plus::util::cli::{
     ConnectedDemux, ConnectedDirect, ConnectedSink, ConnectedSource, ConnectedTagged, HydroCLI,
 };
-use hydroflow_plus::HfBuilder;
+use hydroflow_plus::GraphBuilder;
 use stageleft::{q, Quoted, RuntimeData};
 use syn::parse_quote;
 
 use super::HydroflowPlusMeta;
 
 pub struct CLIRuntime {}
 
-impl<'a> HfDeploy<'a> for CLIRuntime {
+impl<'a> Deploy<'a> for CLIRuntime {
```

Comment:
nit, missed this before, but technically should be `CliRuntime`

Reply:
Gonna rename all the CLI references to Deploy soon, so will leave this to a separate PR.